Cakefile

```diff
@@ -35,6 +35,11 @@ build = (callback) ->
             console.error err
         else
             build_webpack () ->
+                jsPath = build_dir + '/APIBlueprintImporter.js'
+                js = fs.readFileSync(jsPath, 'utf-8')
+                js = js.replace('var Module={"ready"', 'var Module={"ENVIRONMENT":"WEB","ready"')
```

Comment:
Unfortunately I didn't find an easy way to make emscripten runtime believe it is `WEB`. Emscipten documentation mentions you can set `Module.ENVIRONMENT` to force (https://kripken.github.io/emscripten-site/docs/api_reference/module.html#overriding-execution-environment). However in https://github.com/apiaryio/drafter.js/blob/master/src/pre.js we are setting `Module` to an object and thus we can't add to it afterwards.

I think we will need to do one of the following:

- Improve emscripten so it can detect environment from inside WebKit JavaScriptCore as WEB.
- Improve drafter.js so it doesn't override existing `Module` and instead appends.

/c @w-vi